control/server.py

```diff
@@ -112,14 +109,13 @@ def serve(self):
     def _add_server_listener(self):
         """Adds listener port to server."""
 
-        enable_auth = self.config.getboolean("gateway", "enable_auth")
-        gateway_addr = self.config.get("gateway", "addr")
-        gateway_port = self.config.get("gateway", "port")
-        if enable_auth:
+        addr = "{}:{}".format(self.config.gateway.addr, self.config.gateway.port)
```

Comment:
And here I'd do the f-string:
```suggestion
        addr = f"{self.config.gateway.addr}:{self.config.gateway.port}"
```